src/app/parcels/ActionBar/ActionModals/SlotChangeModal.tsx

```diff
@@ -31,11 +42,43 @@ const SlotChangeInput: React.FC<SlotInputProps> = ({ packingSlotsLabelsAndValues
                 onChange={(event) => {
                     setSlot(event.target.value);
                 }}
+                focusOnDropdown={true}
             />
         </>
     );
 };
 
+const SlotChangeModalContent: React.FC<ContentProps> = ({
+    displayModal,
+    packingSlots,
+    setSlot,
+    selectedParcels,
+    warningMessage,
+    onClose,
+    onSlotSubmit,
+}) => {
+    return (
+        displayModal && (
```

Comment:
can we move the boolean check for `displayModal` outside the content and include it with the `actionComplete` check?